.github/workflows/prepare-tanssi-relay-binary.yml

```diff
@@ -0,0 +1,111 @@
+name: Prepare Optimized Dancelight Binary Draft (internal only)
+
+# The code (like generate-release-body) will be taken from the tag version, not master
+on:
+  workflow_dispatch:
+    inputs:
+      sha:
+        description: full sha to build the binary from
+        required: true
+
+jobs:
+  ####### Building binaries #######
+
+  build-binary:
+    runs-on: bare-metal
+    strategy:
+      matrix:
+        cpu: ["", "skylake"]
+    env:
+      RUSTFLAGS: "-C target-cpu=${{ matrix.cpu }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.sha }}
+      - name: Setup Rust toolchain
+        run: rustup show
+      - name: Build Node
+        run: cargo build --profile=production --all
```

Comment:
cargo build --profile=production -p tanssi-relay

Reply:
Thanks for pointing this one out... missed it updated in https://github.com/moondance-labs/tanssi/pull/696/commits/7d82c760c632a4fdd7e9030bd7f51b88db621e9e